assets/js/src/navigation.js

```diff
@@ -129,9 +130,9 @@ function initEachNavToggleSubmenu( nav ) {
 			}
 		});
 
-		submenu.parentNode.classList.add( 'menu-item--has-toggle' );
+		SUBMENUS[ i ].parentNode.classList.add( 'menu-item--has-toggle' );
```

Comment:
Can we make all array usage match? `SUBMENUS[i]`

Reply:
Done.